stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/state/messages/composer/MessageInputState.kt

```diff
@@ -13,11 +13,13 @@ import io.getstream.chat.android.common.state.ValidationError
  * @param action The currently active [MessageAction].
  * @param validationErrors The list of validation errors.
  * @param mentionSuggestions The list of users that can be used to autocomplete the mention.
+ * @param cooldownTimer The amount of time left until the user is allowed to send the next message.
  */
 public data class MessageInputState(
     val inputValue: String = "",
     val attachments: List<Attachment> = emptyList(),
     val action: MessageAction? = null,
     val validationErrors: List<ValidationError> = emptyList(),
-    val mentionSuggestions: List<User> = emptyList()
+    val mentionSuggestions: List<User> = emptyList(),
+    val cooldownTimer: Int,
```

Comment:
Would this make sense to be `0` by default? If the users don't have a cooldown timer set up in dashboard?